yarn.lock

```diff
@@ -1520,7 +1520,12 @@
   resolved "https://registry.yarnpkg.com/@types/mocha/-/mocha-9.1.1.tgz#e7c4f1001eefa4b8afbd1eee27a237fee3bf29c4"
   integrity sha512-Z61JK7DKDtdKTWwLeElSEBcWGRLY8g95ic5FoQqI9CMx0ns/Ghep3B4DfcEimiKMvtamNVULVNKEsiwV3aQmXw==
 
-"@types/node@*", "@types/node@>=12.12.47", "@types/node@>=13.7.0":
+"@types/node@*":
+  version "18.7.16"
```

Comment:
it's not a huge deal, but using the 18.x types can sometimes mislead on what APIs are available. Was there a 16.x that got the URL change?

Reply:
looks like latest 16.x works, yeah